examples/tests/disco_class_scan_config/main.tf

```diff
@@ -0,0 +1,72 @@
+terraform {
+  required_providers {
+    transcend = {
+      version = "0.19.0"
+      source  = "transcend.com/cli/transcend"
+    }
+  }
+}
+
+provider "transcend" {
+  url = "https://api.staging.transcen.dental/"
+}
+
+variable "title" {}
+variable "disco_class_scan_config_vars" {
+  type = object({
+    enabled                    = bool
+    type                       = string
+    schedule_frequency_minutes = number
+    schedule_start_at          = string
+  })
+}
+
+resource "transcend_data_silo" "silo" {
+  type            = "amazonDynamodb"
+  title           = var.title
+  skip_connecting = true
+}
+
+resource "transcend_data_silo_connection" "connection" {
+  data_silo_id = transcend_data_silo.silo.id
+
+  plaintext_context {
+    name  = "role"
+    value = "TranscendAWSIntegrationRole"
+  }
+
+  plaintext_context {
+    name  = "accountId"
+    value = "590309927493"
+  }
+
+  plaintext_context {
+    name  = "region"
+    value = "eu-west-1"
+  }
+
+  // Enable item-level access
+  plaintext_context {
+    name  = "database"
+    value = "true"
+  }
+}
```

Comment:
This is all copied from [content_classification_plugin](https://github.com/transcend-io/terraform-provider-transcend/blob/main/examples/tests/content_classification_plugin/main.tf)